API/projectClass/projectClass.m

```diff
@@ -1034,14 +1034,52 @@ function delete(obj)
         % ----------------------------------------------------------------
         % Editing of Contrasts Block
         
-        function obj = addContrast(obj, varargin)
-            % Adds a new contrast parameter. Expects a parameter name, and with 
-            % key-value pairs with one or more of the following "data",
-            % "background", "bulk in", "bulk out", "scalefactor",
-            % "resolution", "resample", "model"
-            % 
-            % project.addContrast('contrast 1', 'bulkIn', 'Silicon');
-            obj.contrasts.addContrast(obj.getAllAllowedNames(), varargin{:});
+        function obj = addContrast(obj, options)
```

Comment:
It's all written down somewhere . . .

Reply:
Don't understand what is written down somewhere? `addContrast` has always been in `projectClass`

Reply:
Sorry, I'd gone a bit mad by this point. I just mean that I've written the comments elsewhere, i.e., the same points as the other `addContrast` docstrings.